user_data/strategies/MultiTargetClassifierTestStrategy.py

```diff
@@ -0,0 +1,244 @@
+import logging
```

Comment:
we shouldn't be having files in user_data. If a user happens to have a file with the same name, this would cause an update conflict,  many users inexperienced with git will not be able to fix on their own.

The name suggests that it's a test strategy (used for unittests??) - so `tests/strategy/strats/` might be an appropriate location ...

Reply:
I'll remove these. Was adding to PR so testing of a dummy strategy would be easier by the reviewer. But this makes sense.